### ISSUE TITLE
How to label new ocr dataset with square metres

### ISSUE THREAD

**Issue by ahbon123** [keras-team/keras-io#545]
Hello,

I have a new dataset which similar to your training data, but a little bit different. I need to recognize the numbers from the images.

My question is how to label data, for the following images, should I name them as `10240㎡.png` or `10240m2.png`, `9272㎡.png` or `9272m2.png`? Thanks at advance.

![10240㎡](https://user-images.githubusercontent.com/15428667/125152099-11291a00-e17d-11eb-8c13-e1dd453ae248.png)

![9272㎡](https://user-images.githubusercontent.com/15428667/125152027-ca3b2480-e17c-11eb-9b62-10efa63a6139.png)

When I label with `㎡`, I get:

![image](https://user-images.githubusercontent.com/15428667/125152300-59950780-e17e-11eb-8751-a6816ca4f46a.png)

**Comment by sachinprasadhs** [comment]
Hi, You can label either `10240m2.png` or using caret symbol `^` as `10240m^2.png`
